packages/module/patternfly-docs/content/extensions/virtual-assistant/examples/ChatbotMessage/ChatbotMessage.md

```diff
@@ -1,12 +1,73 @@
 ---
 # Sidenav top-level section
 # should be the same for all markdown files
-section: PatternFly AI
+section: PatternFly-AI
 subsection: Chatbot
 # Sidenav secondary level section
 # should be the same for all markdown files
 id: Chatbot messages
 # Tab (react | react-demos | html | html-demos | design-guidelines | accessibility)
 source: react
+propComponents: [
+'Message',
+'PreviewAttachment',
+'AttachmentEdit',
+'ActionProps'
+]
 ---
 
+import Message from '@patternfly/virtual-assistant/dist/dynamic/Message';
+import PreviewAttachment from '@patternfly/virtual-assistant/dist/dynamic/PreviewAttachment';
+import AttachmentEdit from '@patternfly/virtual-assistant/dist/dynamic/AttachmentEdit';
+import customImage from './custom_user_img.jpeg';
+import { RobotIcon } from '@patternfly/react-icons/dist/esm/icons/robot-icon';
+
+## Messages
+
+The `content` prop of the `<Message>` component is passed to a `<Markdown>` component from [react-markdown](https://remarkjs.github.io/react-markdown/). `<Markdown>` is configured to translate plain text strings into PatternFly [`<Content>` components](/components/content) and code blocks into PatternFly [`<CodeBlock>` components.](/components/code-block)
```

Comment:
The Content and CodeBlock component links are broken here.

Reply:
they will be broken until they are pulled into org